modular_zubbers/code/game/objects/items/storage/belt.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/storage/belt/security/Initialize(mapload)
+	. = ..()
+	atom_storage.max_slots = 5
+	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
+	atom_storage.set_holdable(list(
+		/obj/item/ammo_box,
+		/obj/item/ammo_casing/shotgun,
+		/obj/item/assembly/flash/handheld,
+		/obj/item/clothing/glasses,
+		/obj/item/clothing/gloves,
+		/obj/item/flashlight/seclite,
+		/obj/item/food/donut,
+		/obj/item/grenade,
+		/obj/item/gun, //SKYRAT EDIT ADDITION
+		/obj/item/holosign_creator/security,
+		/obj/item/knife/combat,
+		/obj/item/melee/baton,
+		/obj/item/radio,
+		/obj/item/reagent_containers/spray/pepper,
+		/obj/item/restraints/handcuffs,
+		/obj/item/restraints/legcuffs/bola,
+		/obj/item/stock_parts/power_store/cell/microfusion, //SKYRAT EDIT ADDITION
+	))
+	atom_storage.open_sound = 'sound/items/handling/holster_open.ogg'
+	atom_storage.open_sound_vary = TRUE
+	atom_storage.rustle_sound = null
```

Comment:
This is all redundant copy paste, this whole thing can just be your `cant_hold` line added to the original `/obj/item/storage/belt/security/Initialize()` with a bubber comment

Reply:
Done